pkg/ovncontroller/daemonset.go

```diff
@@ -32,6 +38,35 @@ func DaemonSet(
 	runAsUser := int64(0)
 	privileged := true
 
+	volumes := GetVolumes(instance.Name, instance.Namespace)
+	commonVolumeMounts := []corev1.VolumeMount{}
+
+	// add CA bundle if defined
+	if instance.Spec.TLS.CaBundleSecretName != "" {
+		volumes = append(volumes, instance.Spec.TLS.CreateVolume())
+		commonVolumeMounts = append(commonVolumeMounts, instance.Spec.TLS.CreateVolumeMounts(nil)...)
+	}
+
+	ovnControllerVolumeMounts := append(GetOvnControllerVolumeMounts(), commonVolumeMounts...)
+
+	// add OVN dbs cert and CA
+	var ovnControllerTLSArgs []string
+	if instance.Spec.TLS.Enabled() {
```

Comment:
Wondering for setting these client config shouldn't we check if tls enabled for ovndbs? or that's too much

Reply:
you mean check that it's `ssl:...`? Not sure if it would do anything. Ovndbs should be reconciling at the same time so it would probably just add another reconcile loop here. Can follow-up with a patch to do this later maybe if it solves a real problem

Reply:
Actually I was thinking of a case where tls flag get's enabled for northd/ovncontroller but not for ovndbs. May be not a real use case going forward?

Reply:
ah, ok, openstack-operator is managing all of this so not a real use case

Reply:
i.e https://github.com/openstack-k8s-operators/openstack-operator/compare/main...olliewalsh:openstack-operator:ovn_tls#diff-6dd9f181e1fc64412b7bb7a039ccfa76b7250b3d55409457fa8eb290372a4be2